src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -244,8 +244,8 @@
     <comment>Header for a control to toggle if the app should always show the tabs (similar to a website browser).</comment>
   </data>
   <data name="Globals_AlwaysShowTabs.HelpText" xml:space="preserve">
-    <value>When disabled, the tab bar will appear when a new tab is created.</value>
-    <comment>A description for what the "always show tabs" setting does. Presented near "Globals_AlwaysShowTabs.Header".</comment>
+    <value>When disabled, the tab bar will appear when a new tab is created. Can not be disabled while Hide Title Bar is On.</value>
+    <comment>A description for what the "always show tabs" setting does. Presented near "Globals_AlwaysShowTabs.Header". This setting cannot be disabled while "Globals_ShowTitlebar.Header" is enabled.</comment>

```

Comment:
I don't think the change to the comment is necessary, especially since we're probably misusing the comment field anyways. It's only there to help translators and our current description probably doesn't help at all, considering that no translator in the world will know what a "Globals_AlwaysShowTabs.Header" is in the first place. 😅

Reply:
FWIW, I recommended it.

I think there is value in the comment indicating what the localization context is. There's no way for us to say "Can not be disabled while $(RS:Globals_ShowTitlebar.Header) is enabled", so the next best thing is telling the translator that.

I might suggest a reword, however:

1. `can not` -> `cannot`
2. "The name `Hide Title Bar` should match the translated value for `Globals_ShowTitlebar.Header`." Putting it in quotes inside the `value` might make it even more obvious that it's meant to be transcluded

Reply:
Want me to push those updates?